docker/app.Dockerfile

```diff
@@ -65,6 +65,7 @@ RUN echo "Installing XDebug 3 (in disabled state)" \
     && echo "xdebug.start_with_request=yes" >> /usr/local/etc/php/conf.d/disabled/docker-php-ext-xdebug.ini \
     && echo "xdebug.client_host=host.docker.internal" >> /usr/local/etc/php/conf.d/disabled/docker-php-ext-xdebug.ini \
     && echo "xdebug.client_port=9003" >> /usr/local/etc/php/conf.d/disabled/docker-php-ext-xdebug.ini \
+    && echo "xdebug.max_nesting_level=512" >> /usr/local/etc/php/conf.d/disabled/docker-php-ext-xdebug.ini \
```

Comment:
@jasonbahl this gets us past the xdebug inifinite loop failure that is reached.  Doesn't solve a potential problem in all the foreach loops with recursive apply_filters in the instrumentSchema code, but tests pass now.  :shrug: